README.md

```diff
@@ -364,6 +364,17 @@ The `safe` options flag set to `true` enables this behavior:
 const expanded = await jsonld.expand(data, {safe: true});
 ```
 
+### Using with React Native
+
+Using this library with React Native requires the
+[`data-integrity-rn`](https://github.com/digitalcredentials/data-integrity-rn)
```

Comment:
Seems an odd name for the polyfill. I wonder if there's a more general name like `polyfill-rn`? It's scoped to `@digitalcredentials` so may be that would be ok?

Hmm, maybe we also want to say that any polyfill will do, such as this one? That way we don't have a defacto dependency that we have to check with when doing updates.

Reply:
Rephrased to say 'a polyfill such as', which hopefully captures the same thought.